src/events/components/Attendance/AttendButton.tsx

```diff
@@ -55,6 +55,20 @@ const AttendButton: FC<IAttendButtonProps> = (props: IAttendButtonProps) => {
       toggleModal={toggleModal}
       setCaptcha={signUp}
       errorText={FAILED_CAPTCHA_ERROR_TEXT}
+      onError={async (error) => {
+        const user = await getUser();
+        Sentry.captureEvent({
+          message: 'User failed turnstile challenge when signing up for event.',
+          level: Sentry.Severity.Fatal,
```

Comment:
Det er ikke riktig nivå, dette er maks en error, sannsynligvis bare warning

Reply:
I utgangspunktet er jeg enig i at det burde være warning, men vi vet at i 99.9% av tilfellene som dette skjer så er det ikke meningen at det skal ha skjedd. For hvorfor skal folk gidde å botte når det ikke funker uansett siden vi har turnstile. Så mener det er error i vårt tilfelle her.

Reply:
Håpet er ihvertfall at turnstile gjør det vanskelig nok å botte til at ingen gidder. Men har ikke testet selv

Reply:
> 99.9% av tilfellene som dette skjer så er det ikke meningen at det skal ha skjedd

Nei, det er ikke det vi vet.

Dette er ikke en fatal error, og skal ikke behandles som det, det er helt naturlig at Turnstile feiler, det at den noen ganger feiler betyr at den er "working as intended".

Reply:
Enig i at det ikke er en fatal error. Synes det kan være error sett situasjonen vår.

Mener du det er "helt naturlig" om turnstile i blant feiler når reelle brukere prøver å melde seg på?

Reply:
> "helt naturlig" om turnstile i blant feiler når reelle brukere prøver å melde seg på

Ja, fordi bot-deteksjon er et vanskelig problem, og jeg antar at de nødvendigvis må ha noen false-positives. Nå er jeg også ikke helt kjent med API-et til turnstile her og det kan godt være at det at den sier feil en gang bare betyr at brukeren må trykke på knappen en gang til eller må flytte på musepekeren litt til for å samle info, idk.

Vi vet jo allerede fra stats at den feiler _ganske ofte_, og kan være måten vi har satt den opp (med at den bare er i en modal) legger opp til at den feiler oftere enn nødvendig.

Reply:
Ok sure kjører warning. Men synes personlig det like godt kunne vært error i vårt tilfelle.